lib/parser.js

```diff
@@ -85,9 +85,15 @@ function parseTag(html, onTag, escapeHtml) {
           tagStart = false;
           continue;
         }
-        if ((c === '"' || c === "'") && html.charAt(currentPos - 1) === "=") {
-          quoteStart = c;
-          continue;
+        if ((c === '"' || c === "'")) {
+          var i = 1;
+          while ((html.charAt(currentPos - i) === " ") || (html.charAt(currentPos - i) === "=")) {
```

Comment:
A temporary variable is needed to simplify the code here, such as `var ic = html.charAt(currentPos - i)`